lib/import/tabular_basic.rb

```diff
@@ -0,0 +1,37 @@
+module Import
+  module TabularBasic
+    private 
+      # valid header fields
+      def parse_headers(row)
+        headers = []
+        $i = 0
```

Comment:
@mcritchlow There is no enumerable method `row.each` for Excel. It only define the`[](ind)` method that references the the value at cells[ind].